docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -251,7 +251,7 @@ The following table provides the API details associated with each account-level
 
 :::note
 
-Account Global Admin has Namespace Admin permissions on Namespaces.
+Account Owners and Global Admins have Namespace Admin permissions on Namespaces.
```

Comment:
suggestion: add "permissions on all Namespaces"

Reply:
Adopting as suggested. Thank you.